src/components/DownloadContactListButton.tsx

```diff
@@ -1,33 +1,39 @@
-import React from 'react'
-import { UnprocessedResultsFromCRM } from '../types'
 
+import React from 'react'
+import { UnprocessedResultsFromCRM, OwnerType } from '../types'
 type DownloadButtonProps = {
     results: UnprocessedResultsFromCRM[]
 }
+
 export function DownloadContactListButton (props: DownloadButtonProps) {
     let downloadUrl = null
     let csvData = '"Property Address","Property Type (Marketing)","Owner","Owner Mobile","Owner Phone","Contact","Contact Mobile","Contact Work Phone"\r\n'
-
-    props.results.forEach((result) => {
+    const arrayOfPropertyObjects = props.results
+    function getUniqueListBy (arr: UnprocessedResultsFromCRM[], key: string) {
```

Comment:
was there an issue with duplicates? I thought I handled that in the deluge code that fetches unique properties